build.gradle.kts

```diff
@@ -154,6 +155,18 @@ publishing {
     }
 }
 
+nexusPublishing {
+    packageGroup.set(project.group.toString())
+    repositories {
+        sonatype {
+            nexusUrl.set(uri("https://s01.oss.sonatype.org/service/local/"))
+            snapshotRepositoryUrl.set(uri("https://s01.oss.sonatype.org/content/repositories/snapshots/"))
+            username.set(System.getenv("MAVEN_CENTRAL_STAGING_REPO_USER"))
+            password.set(System.getenv("MAVEN_CENTRAL_STAGING_REPO_PASSWORD"))
```

Comment:
It looks like if we set the environment variable names in the right way, this will happen automatically: https://github.com/gradle-nexus/publish-plugin#publishing-to-maven-central-via-sonatype-ossrh

> ORG_GRADLE_PROJECT_sonatypeUsername and ORG_GRADLE_PROJECT_sonatypePassword environment variables.

Reply:
You are right, seems to work.